src/pykeen/models/unimodal/kg2e.py

```diff
@@ -78,7 +70,11 @@ def __init__(
         r"""Initialize KG2E.
 
         :param embedding_dim: The entity embedding dimension $d$. Is usually $d \in [50, 350]$.
-        :param dist_similarity: Either 'KL' for Kullback-Leibler or 'EL' for expected likelihood. Defaults to KL.
+        :param dist_similarity:
+            The similarity measures for gaussian distributions. Defaults to
+            :class:`~pykeen.nn.sim.NegativeKullbackLeiblerDivergence`.
```

Comment:
what does the ~ do inside the class link?

Reply:
The rendered documentation contains only the last part (`NegativeKullbackLeiblerDivergence`) instead of the fully qualified path. I think this makes the text more readable (and you can still click on the link to get to the actual path).

Reply:
cf. https://www.sphinx-doc.org/en/master/usage/referencing.html#cross-referencing-syntax